dmoj/cptbox/helper.cpp

```diff
@@ -103,7 +131,32 @@ int cptbox_child_run(const struct child_config *config) {
 
     if (ptrace_traceme())
         return 204;
+
     kill(getpid(), SIGSTOP);
```

Comment:
Do we want to debug the seccomp initialization with `ptrace`?

Reply:
No trap will be generated until `seccomp` returns when `ptrace` is set up with `PTRACE_O_TRACESECCOMP`. If we move the `SIGSTOP` `kill` until after the `seccomp` initialization, then `cptbox` will receive a syscall trap (for `kill`) before being signaled that everything is "ready" by `SIGSTOP`.

I don't really want to reason if that's OK.